app-misc/freeplane/metadata.xml

```diff
@@ -2,12 +2,9 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<maintainer type="person">
-		<email>xmw@gentoo.org</email>
-		<name>Michael Weber</name>
+		<email>rtgiskard@gmail.com</email>
```

Comment:
Have you talked with @xmw so he's okay with this? Also, who's going to commit for you? You need to add either a developer, or project, in the metadata.xml to commit for you. 

You also can't claim packages from other people without their consent. I don't see a bug linked, or xmw's approval in this PR.